.github/workflows/xcode_check.yml

```diff
@@ -0,0 +1,24 @@
+name: Check xcode completeness
+
+on:
+  push:
+    paths:
+    - 'source/**'
+    - 'tests/**'
+  pull_request:
+    paths:
+    - 'source/**'
+    - 'tests/**'
+
+jobs:
+  check-xcode-completeness:
```

Comment:
What do you think about adding one of these checks for Code::Blocks, too? It's more accessible, but I'm not the only one who uses something else.

It'd be nice to have both project file checks in the same action, but running both when either one fails and having the steps named well seems like it might not be fun

Reply:
Sounds like a good idea, I'm in favor of checking such project files automatically.

But can we keep that for a separate PR? I currently don't know what exactly needs to be done for Code::Blocks, while I do see issues with XCode projects in combination with PRs that add new files by developers that don't have a Mac.